lib/gitgitgadget.ts

```diff
@@ -144,7 +143,7 @@ export class GitGitGadget {
             throw new Error(`User ${vouchingUser} lacks permission for this.`);
         }
 
-        if (!this.isUserAllowed(user)) {
+        if (!this.allowedUsers.delete(user)) {
```

Comment:
It did take me a couple of moments to wrap my head around why this is correct, but yes, it is!